### PR TITLE
submit draft July TLC minutes review and approval

### PR DESCRIPTION
**Pull request by jonathanwheeler01**
Submitting the minutes of the July 10/11 TLC meeting for review. Approval can be provided here or in the TLC slack channel.

### CODE REVIEW

File: leadership_minutes/2024/2024-07-11-leadership-minutes.md [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+Notetaker: Jon
+
+## Agenda and notes
+
+1.  Update on action items
+    a.  Nathaniel will look up info from last year's trainer active status renewal process. By the next meeting.
+	    1. From 5 Dec 2023 notes on renewal:
+		    1.  Most valuable alternative activities included Trainers Leadership and IT curriculum maintenance.
+		    2.  4 demos could substitute for 1 Instructor Training.
+		    3. 2 Instructor Training could cover other requirements (certainly all demos).
```
Comment (ndporter) [inline leadership_minutes/2024/2024-07-11-leadership-minutes.md:15]:
```suggestion
		    3. Two Instructor Training events could cover other requirements (certainly all demos).
```

File: leadership_minutes/2024/2024-07-11-leadership-minutes.md [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+		    4. Otherwise, it will be hard to get away from a case-by-case review if we want to keep the "alternative activities" option for sustaining active status.
+		    5. Number of people who have done demos or workshops that weren't included in their summary was alarmingly high. Does this reflect a problem?
```
Comment (ndporter) [inline leadership_minutes/2024/2024-07-11-leadership-minutes.md:17]:
```suggestion
		    5. A number of people who have done demos or workshops that weren't included in their summary was alarmingly high. Does this reflect a problem?
```

File: leadership_minutes/2024/2024-07-11-leadership-minutes.md [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+			    1. Demos are recorded in AMY when host submits results (maybe? Nathaniel will check with Core Team).
```
Comment (ndporter) [inline leadership_minutes/2024/2024-07-11-leadership-minutes.md:18]:
```suggestion
			    1. Demo hosts are recorded in calendly and demo host form, with records maintained and provided for renewal process by Maneesha (not recorded in AMY)
```

**Review by ndporter — COMMENTED** [review]
Looks good overall. Updated a couple lines for clarity and added answer I verified with Sher about documenting teaching demos.

**Comment by ndporter** [comment]
Requesting review from a 3rd party on leadership as @jonathanwheeler01 and I both contributed. Once approved, it is ready to merge and has had its month elapse.

**Review by ragamouf — APPROVED** [review]
Happy to approve minor grammatical changes

**Pull request merged**